### PR TITLE
Add documentation for cluster down read config

### PR DESCRIPTION
**Pull request by madolson**
See https://github.com/antirez/redis/pull/6144

### CODE REVIEW

File: topics/cluster-tutorial.md [hunk 1]
```diff
@@ -224,6 +224,8 @@ as you continue reading.
...
 * **cluster-migration-barrier `<count>`**: Minimum number of slaves a master will remain connected with, for another slave to migrate to a master which is no longer covered by any slave. See the appropriate section about replica migration in this tutorial for more information.
 * **cluster-require-full-coverage `<yes/no>`**: If this is set to yes, as it is by default, the cluster stops accepting writes if some percentage of the key space is not covered by any node. If the option is set to no, the cluster will still serve queries even if only requests about a subset of keys can be processed.
+* **cluster-allow-reads-when-down `<yes/no>`**: If this is set to no, as it is by default,, a node in a Redis Cluster will stop serving all traffic when when the cluster is marked as fail, either when a node can't reach a quorum of masters or full coverage is not met. This prevents reading inconsistent data from a node that is unaware of changes in the cluster. This option can be set to yes to allow reads from a node during these events, which is useful for applications that want to prioritize read availability data but want to prevent inconsistent writes.
```
Comment (itamarhaber) [inline topics/cluster-tutorial.md]: One comma is sufficient.

Comment (itamarhaber) [inline topics/cluster-tutorial.md]: As this is a "new" feature, please add the "as of version x.y" once determined.

**Comment by madolson** [comment]
Something accepted without pinging, this is a well maintained repo :)

**Comment by itamarhaber** [comment]
🤩

**Pull request merged**